go/libraries/doltcore/type_conversion.go

```diff
@@ -145,6 +151,10 @@ func convStringToFloat(val types.Value) (types.Value, error) {
 		return nil, nil
 	}
 
+	if val.Equals(types.String("\"\"")) {
```

Comment:
If the value in the csv is """""" then the val should be "\"\"" I don't think we want this to be Null.  It seems like that should be an error.  The same goes for the ones below.